app/scripts/inpage.js

```diff
@@ -35,8 +35,17 @@ import log from 'loglevel';
 import { v4 as uuid } from 'uuid';
 import { WindowPostMessageStream } from '@metamask/post-message-stream';
 import { initializeProvider } from '@metamask/providers/initializeInpageProvider';
+// eslint-disable-next-line import/order
 import shouldInjectProvider from '../../shared/modules/provider-injection';
 
+///: BEGIN:ONLY_INCLUDE_IF(build-beta,build-flask)
```

Comment:
wonder if we can replace these with a check against `process.env.MULTICHAIN_API` instead

Reply:
Do we think we need to bracket off the imports behind flask/beta build flags?

We're not doing that for a bunch of stuff in `metamask-controller` that is only used in Multichain API?

I suppose those scripts don't get loaded into memory if we do it this way?